.github/workflows/com-deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: GitHub Pages
```

Comment:
This hasn't been adapted for Antora/AsciiDoc. The tasks done in the file accomplish the same thing as `prod-publish.yml` (which looks to be incorrectly named).